shared/fp-tools.ts

```diff
@@ -0,0 +1,3 @@
+const compose = <A extends unknown, F extends (arg0: A) => A>(...fns: F[]) => (initialVal: A) => fns.reduce((acc, fn) => fn(acc), initialVal);
```

Comment:
we have Ramda lib , and compose fn already[ in it ](https://ramdajs.com/docs/#compose)

Reply:
Thank you!
I have tried to use it. Can you review it again, please?